termino/termino.py

```diff
@@ -55,14 +63,27 @@ async def red_delete_data_for_user(self, **kwargs):
         """Nothing to delete"""
         return
 
+    async def startup(self):
+        await self.bot.wait_until_red_ready()
+        conf = await self.config.all()
+        maybe_channel = conf.get("restart_channel", None)
+        await self.config.restart_channel.clear()
+        # Using the walrus operator we can check if the channel is originally None
+        # Or that we cannot find that channel
+        if maybe_channel is None or (ch := self.bot.get_channel(maybe_channel)) is None:
+            return
+        try:
+            await ch.send(conf["restarted_message"])
+        except discord.Forbidden as e:
+            log.info("Unable to send a confirmation message to the restart channel")
+            log.debug("Unable to send a message", exc_info=e)
+
     async def confirmation(self, ctx: commands.Context, _type: str):
         await ctx.send(f"Are you sure you want to {_type} {ctx.me.name}? (yes/no)")
         with contextlib.suppress(asyncio.TimeoutError):
             pred = MessagePredicate.yes_or_no(ctx, user=ctx.author)
             await ctx.bot.wait_for("message", check=pred, timeout=60)
-            if pred.result:
-                return True
-            return False
+            return pred.result
```

Comment:
I like this, thanks. Sometimes I do stupid stuff like this when it will return a bool anyway :P